### PR TITLE
Show receipts in the settings

### LINKED ISSUE #1706: Allow users to download a receipt as PDF
Subsequent to https://github.com/ChainSafe/ui-monorepo/pull/1492 users should be able to download their invoice as pdf.
This requires back-end pdf generation

Make API call to request PDF for a specific Invoice ID
Download the PDF to device, and allow browser to save.

### LINKED ISSUE #1808: Show the latest bills and allow to download them from the subscription tab
Follow the mock
image

Initiating payment on an unpaid invoice is out of the scope of this issue.

### PR DESCRIPTION
**Pull request by Tbaut**
closes #1808
closes #1706

I show the invoice uuid now because there's no way to DL an invoice. This doesn't look awesome on mobile because of this.

---
Submission checklist: 

   

#### Layout
- [x] Change looks good in the desktop web ui
- [x] Change looks good in the mobile web ui

#### Theme
- [x] Components / elements inspected in light mode 
- [x] Components / elements inspected in dark mode

### CODE REVIEW

**Review by FSM1 — APPROVED** [review]

File: packages/files-ui/src/Components/Elements/InvoiceLines.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          >
+            <Trans>No invoice found</Trans>
+          </Typography>
+        </div>
+      )}
+      {!!invoicesToShow?.length && (
+        invoicesToShow.map(({ paid_on, amount, currency, uuid }, index) =>
+          <section
+            className={classes.setOption}
+            key={index}
```
Comment (FSM1) [inline packages/files-ui/src/Components/Elements/InvoiceLines.tsx]:
```suggestion
            key={uuid}
```

**Review by FSM1 — APPROVED** [review]
One tiny suggestion, but overall looking great

File: packages/files-ui/src/Components/Elements/InvoiceLines.tsx (hunk 1 above)
Comment (Tbaut) [inline packages/files-ui/src/Components/Elements/InvoiceLines.tsx]:
```suggestion
        invoicesToShow.map(({ paid_on, amount, currency, uuid }) =>
          <section
            className={classes.setOption}
            key={uuid}
```

**Review by tanmoyAtb — APPROVED** [review]

**Comment by Tbaut** [comment]
We can properly test this when this is merged https://github.com/ChainSafe/files-api/pull/1758

**Pull request merged**